META6.json

```diff
@@ -16,7 +16,7 @@
     "Pod::Load:ver<0.5.4+>",
     "Pod::To::HTML:ver<0.7.1+>:auth<github:Raku>",
     "Pod::From::Cache:ver<0.2+>",
-    "Perl6::TypeGraph:ver<2.1.3+>:auth<github:antoniogamiz>:api<2>",
+    "Doc::TypeGraph:ver<2.1.3+>:auth<github:antoniogamiz>:api<2>",
```

Comment:
I think the `auth` changed to `github:jj` ?

Reply:
Fixed, thanks!